R/multivariate_pipeline.R

```diff
@@ -60,38 +53,26 @@ run_multivariate_pipeline <- function(
     return(y)
   }
 
-  filter_cv_data <- function(y, maf, X_scalar, y_scalar, dropped_sample, mrmash_weights_prior_matrices,
-                             mrmash_weights_prior_matrices_cv) {
-    drop_indx <- which(colSums(is.na(y)) == nrow(y))
-
-    if (length(drop_indx) >= 1) {
-      y <- y[, -drop_indx]
-      maf <- maf[-drop_indx]
-      X_scalar <- X_scalar[-drop_indx]
-      y_scalar <- y_scalar[-drop_indx]
-      dropped_sample <- lapply(dropped_sample, function(x) x[-drop_indx])
-
+   filter_prior_matrices <- function(
```

Comment:
In this case it is more than just filter prior. It's construct it as well. Please change `filter_prior_matrices` function name to `initialize_multivariate_prior`